lib/vertica/protocol/frontend/startup.rb

```diff
@@ -8,12 +8,22 @@ def initialize(user, database, options = nil)
         @user     = user
         @database = database
         @options  = options
+        @type     = "vertica-rb"
+        @pid      = Process.pid.to_s
+        @platform = Gem::Platform.local.to_s
```

Comment:
I prefer simply using `RUBY_PLATFORM` here. Simpler, and theoretically the `Gem` constant may not be loaded (unlikely though).